lib/ecto/changeset.ex

```diff
@@ -2067,6 +2065,41 @@ defmodule Ecto.Changeset do
     end
   end
 
+  @doc """
+  Returns a list of fields that are present.
```

Comment:
```suggestion
  Returns which fields in a list would pass required validations (the fields that are present).
```

I am worried that the name is not obvious, given your original commit. The fact we don’t mention “required” in the description can also be considered a flag. Is this description clearer? We could call it `filter_present` but I don’t want to introduce this terminology to the API. Thoughts?

Reply:
Your wording is definitely better.

I wouldn't want to introduce "present" into the function names either. I was struggling to think of a clearer function name though. It wasn't the most intuitive to me when I first read it.

The only other ones I could think of were `keep_required` or `take_required`. Maybe they make the intent clearer but I don't know if `keep` is a verb used in Elixir and `take` is associated with maps iirc.